pkg/client/functions.go

```diff
@@ -140,10 +141,11 @@ func getRunSwaggerError(err error) error {
 }
 
 // ListRuns lists all the available results from previous function runs
-func (c *DefaultFunctionsClient) ListRuns(ctx context.Context, organizationID string) ([]v1.Run, error) {
+func (c *DefaultFunctionsClient) ListRuns(ctx context.Context, organizationID string, since *int64) ([]v1.Run, error) {
```

Comment:
Does it have to be a pointer? would 0 be an ok default value? Also it seems `time.Duration` or `time.Time` would be more appropriate here.